modules/lsp/default.nix

```diff
@@ -88,7 +88,7 @@ in
             ''
               vim.lsp.config(${luaName}, ${luaCfg})
             ''
-            + lib.optionalString server.activate ''
+            + lib.optionalString (server.activate && server.name != "*") ''
```

Comment:
We should move this to the `activate` option's default, and also add a literal expression `defaultText`:


In modules/LSP/server.nix:
```nix
activate = mkOption {
  # ...
  default = config.name != "*";
  defaultText = lib.literalExression ''
    config.name != "*"
  '';
};
```

Reply:
Done!